saleor/order/actions.py

```diff
@@ -83,31 +87,68 @@ def order_created(
 ):
     events.order_created_event(order=order, user=user, app=app, from_draft=from_draft)
     manager.order_created(order)
-    payment = order.get_last_payment()
-    if payment:
-        if order.is_captured():
-            order_captured(
+    payments = get_active_payments(order)
+    if payments:
+        authorized_payments = [
+            p for p in payments if p.charge_status == ChargeStatus.AUTHORIZED
+        ]
+        if authorized_payments:
+            order_authorized(
                 order=order,
                 user=user,
                 app=app,
-                amount=payment.total,
-                payment=payment,
+                amount=authorized_payments[0].total,
+                payment=authorized_payments[0],
                 manager=manager,
             )
-        elif order.is_pre_authorized():
-            order_authorized(
+
+        captured_payments = [
+            p
+            for p in payments
+            if p.charge_status
+            in [
```

Comment:
It would be useful to have a helper like `get_captured_payments(payments)` that we can easy reuse and it encapsulates the logic of checking proper statuses, do we have something like that?